tests/test_default_router.py

```diff
@@ -137,7 +137,6 @@ def test_collision(self, template):
         )
 
     @ddt.data(
-        '/repos/{org}/{repo}/compare/{simple-vs-complex}',
```

Comment:
Separately, it would be good to create some new tests that expect (and assert) that ValueError is raised when an invalid field name is specified in the template.